popup.html

```diff
@@ -10,8 +10,9 @@
 <body>
     <div class="spinner"></div>
     <div class="linkList">
-        <div>
+        <div class="buttonContainer">
             <button class="saveButton" type="button">Save Tab</button>
```

Comment:
Add and id for "saveButton" and switch **. saveButton** to use **#saveButton**

Right now the click event will be bound to both buttons